db/migrate/20170711235422_create_articles.rb

```diff
@@ -0,0 +1,14 @@
+ROM::SQL.migration do
+  change do
+    create_table :articles do
+      primary_key :id
+      column :slug, :text, null: false, unique: true, index: true
+      column :title, :text, null: false
+      column :body, :text, null: false
+      column :published, :boolean, default: false, null: false, index: true
```

Comment:
@JuanitoFatas I was actually thinking there's still merit in having both columns. For instance, if you wanted to unpublish something but still preserve its original publishing date, you'd need both the boolean and the timestamp as separate columns. Does that seem reasonable to you?